e2e/defaults.env

```diff
@@ -11,4 +11,4 @@
 export HOME=${HOME:-/home/ubuntu/}
 export TESTDIR=${TESTDIR:-$E2EDIR/tests/${E2ETYPE:-free5gc}}
 export LIBDIR=${LIBDIR:-$E2EDIR/lib}
-export REVISION=${REVISION:-main}
+export REVISION=${REVISION:-v3.0.0}
```

Comment:
This is also broken now for some reason. It defaults to main when 001.sh uses it to pull the workloadcluster pkg.
I tried it using the SHA for the head of the v3.0.0 branch and it deployed the cluster ok.